### ISSUE TITLE
The default selected item and unselected item in the code completion list are the same background color

### ISSUE THREAD

**Issue by immadolf** [catppuccin/jetbrains#9]
this theme:
![图片](https://user-images.githubusercontent.com/12369197/172281494-ece8f06b-fe84-453b-93d6-7cd537e781a3.png)

other theme:
![图片](https://user-images.githubusercontent.com/12369197/172281817-af93f33a-9282-41e5-ac84-1d14feaa1109.png)

**Comment by BButner** [comment]
Trying to reproduce this, are you on macOS? I'm able to reproduce this issue on macOS, but not Windows

**Comment by immadolf** [comment]
yes，i use macOS 12

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Beau Butner ***@***.***>
发送时间: Wednesday, June 15, 2022 11:50:37 PM
收件人: catppuccin/jetbrains ***@***.***>
抄送: immadolf ***@***.***>; Author ***@***.***>
主题: Re: [catppuccin/jetbrains] The default selected item and unselected item in the code completion list are the same background color (Issue #9)


Trying to reproduce this, are you on macOS? I'm able to reproduce this issue on macOS, but not Windows

―

**Comment by BButner** [comment]
Gotcha. I'm able to replicate this so I'll see if I can whip up a fix

**Comment by immadolf** [comment]
Yes, I saw your commit, but I took the liberty of asking how I could test it locally.

**Comment by BButner** [comment]
@immadolf What I did was clone the project, install the Microsoft JDK that it apparently depends on, and then run the `Run Plugin` Gradle task, which builds the plugin from source and runs it in a testing IntelliJ instance.

I did have to specify a newer version of IntelliJ on my MacBook Pro however, as I'm on an M1 Pro and the older version of IntelliJ that's there by default was throwing errors for me unrelated to my change. (Specifically in `gradle.properties` I set `platformVersion` to `platformVersion = 2022.1`.
